xarray/plot/plot.py

```diff
@@ -446,12 +468,13 @@ def newplotfunc(darray, x=None, y=None, figsize=None, size=None,
                           "Use colors keyword instead.",
                           DeprecationWarning, stacklevel=3)
 
```

Comment:
Can you add `rgb` as an actual argument inline, after x and y?

Reply:
I *could*, but this would be a breaking change for anyone using positional arguments.  I'd therefore prefer not to, as it's just as easy to do the breaking bit later.